vulnfeeds/utility/logger/logger.go

```diff
@@ -79,10 +80,13 @@ func log(level slog.Level, msg string, a []any) {
 	runtime.Callers(3, pcs[:]) // skip [Callers, log, Info/Warn/etc]
 	r := slog.NewRecord(time.Now(), level, msg, pcs[0])
 	r.Add(a...)
+	//nolint:errcheck
 	slogLogger.Handler().Handle(context.Background(), r)
 }
 
 // Info prints an Info level log.
+//
+//nolint:contextcheck,nolintlint
```

Comment:
Huh why does this need both ignoring contextcheck, and ignoring nolint at the same time? Seems mutually exclusive

Reply:
without any nolints:
```
cmd/download-cves/main.go:105:16: Function `Warn->log` should pass the context parameter (contextcheck)
                                logger.Warn("Failed to decode NVD data", slog.Any("err", err))
```
with just nolint:contextcheck on `Warn`:
 ```
 utility/logger/logger.go:99:1: directive `//nolint:contextcheck` is unused for linter "contextcheck" (nolintlint)
//nolint:contextcheck
 ```
I can't win :pensive: 

 I think it wants the `//nolint:contextcheck` on the calling function, but that's annoying.
 I added it to the other functions to avoid this in the future.